holidays/locale/en_US/LC_MESSAGES/CH.po

```diff
@@ -0,0 +1,176 @@
+#: holidays/countries/switzerland.py:158 holidays/countries/switzerland.py:168
+#: holidays/countries/switzerland.py:175 holidays/countries/switzerland.py:193
+#: holidays/countries/switzerland.py:206 holidays/countries/switzerland.py:219
+#: holidays/countries/switzerland.py:223 holidays/countries/switzerland.py:247
+#: holidays/countries/switzerland.py:264 holidays/countries/switzerland.py:271
+msgid "Berchtoldstag"
+msgstr "Berchtold's Day"
+
+#. Corpus Christi.
+#: holidays/countries/switzerland.py:106 holidays/countries/switzerland.py:160
+#: holidays/countries/switzerland.py:169 holidays/countries/switzerland.py:187
+#: holidays/countries/switzerland.py:194 holidays/countries/switzerland.py:213
+#: holidays/countries/switzerland.py:230 holidays/countries/switzerland.py:241
+#: holidays/countries/switzerland.py:258 holidays/countries/switzerland.py:265
+msgid "Fronleichnam"
+msgstr "Corpus Christi"
+
+#. Assumption of Mary.
+#: holidays/countries/switzerland.py:109 holidays/countries/switzerland.py:164
+#: holidays/countries/switzerland.py:170 holidays/countries/switzerland.py:188
+#: holidays/countries/switzerland.py:195 holidays/countries/switzerland.py:214
+#: holidays/countries/switzerland.py:234 holidays/countries/switzerland.py:242
+#: holidays/countries/switzerland.py:259 holidays/countries/switzerland.py:266
+msgid "Mariä Himmelfahrt"
+msgstr "Assumption of Mary"
+
+#. All Saints' Day.
+#: holidays/countries/switzerland.py:112 holidays/countries/switzerland.py:152
+#: holidays/countries/switzerland.py:165 holidays/countries/switzerland.py:171
+#: holidays/countries/switzerland.py:189 holidays/countries/switzerland.py:199
+#: holidays/countries/switzerland.py:203 holidays/countries/switzerland.py:215
+#: holidays/countries/switzerland.py:235 holidays/countries/switzerland.py:243
+#: holidays/countries/switzerland.py:260 holidays/countries/switzerland.py:267
+msgid "Allerheiligen"
+msgstr "All Saints' Day"
+
+#. Immaculate Conception.
+#: holidays/countries/switzerland.py:115 holidays/countries/switzerland.py:172
+#: holidays/countries/switzerland.py:190 holidays/countries/switzerland.py:200
+#: holidays/countries/switzerland.py:216 holidays/countries/switzerland.py:236
+#: holidays/countries/switzerland.py:244 holidays/countries/switzerland.py:261
+#: holidays/countries/switzerland.py:268
+msgid "Mariä Empfängnis"
+msgstr "Immaculate Conception"
+
+#. Labour Day.
+#: holidays/countries/switzerland.py:119 holidays/countries/switzerland.py:122
+#: holidays/countries/switzerland.py:159 holidays/countries/switzerland.py:179
+#: holidays/countries/switzerland.py:207 holidays/countries/switzerland.py:220
+#: holidays/countries/switzerland.py:224 holidays/countries/switzerland.py:229
+#: holidays/countries/switzerland.py:272
+msgid "Tag der Arbeit"
+msgstr "Labour Day"
+
+#. Jeûne genevois.
+#: holidays/countries/switzerland.py:134
+msgid "Jeûne genevois"
+msgstr "Jeûne genevois"
```

Comment:
Not really sure:
```suggestion
msgstr "Genevan Fast"
```

At least adds some context.

Reply:
I was guided by [English wiki](https://en.wikipedia.org/wiki/Je%C3%BBne_genevois), although I didn't really like it myself.